CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Draft
 
+## 6.1.4 (17-11-2021)
```

Comment:
@BC-Chkalov-Valerii please keep Draft here as we don't know what version will be released.